### ISSUE TITLE
Normalization before finetuning.

### ISSUE THREAD

**Issue by XinyuanLiao** [vfeofanov/mantis#6]
Hi,

This is an amazing work for time series classification tasks. When I fine-tune the Mantis to my custom dataset, should I do the normalization (e.g., RevIN or min-max normalization) before a adapter or model.fit?

Best regards.

**Comment by vfeofanov** [comment]
Hi, 

For univariate problems, there is no need to apply any normalization as the instance-level normalization is performed during the forward pass. In the multivariate case, if you want to use an adapter, you may need or not need to normalize the channels (variables) depending on your data. Since adapters linearly mix the channels, we normally assume that they have the same unit measures. Hence, if you deal with time series variables of different nature (e.g., current, voltage, temperature), then it is worth trying to apply normalization before passing them to an adapter.

Thanks for the compliment! Hope Mantis will be useful for your project :)
Let me know if you have any other question.

Vasilii

**Comment by XinyuanLiao** [comment]
Thanks for your reply! However, in my experiment, the fine-tune performance became worse after adding a max-abs normalization before the SVD adapter. It's a very interesting result.
